toplevel/coqargs.ml

```diff
@@ -143,7 +141,6 @@ let default_pre = {
   vo_includes  = [];
   vo_requires  = [];
   load_vernacular_list = [];
-  inputstate   = None;
```

Comment:
If we remove this we should also remove `outputstate`; but last time we tried there were significant complaints from users, for example Adam was using it to snapshot; IMHO we should poll the users again.

At some point we should have a better way to do persistent incremental checking, but meanwhile it seems risky to remove this without providing a replacement, for all defects the current hack has.

Reply:
OK thanks for the information. I won't push for this, this was just a clean-up in passing. So I'll restrict the PR to removing `-require`.

Reply:
Sounds good, thanks!

Reply:
But you are right this is a defect of our current approach; we are using `deprecated` to mark stuff with known issues; I think doing this is wrong.

I tried to add the known_issues file but didn't go too far :/